src/algebra/phi-function.md

```diff
@@ -143,6 +143,10 @@ $$a^n \equiv a^{n \bmod \phi(m)} \pmod m$$
 
 This allows computing $x^n \bmod m$ for very big $n$, especially if $n$ is the result of another computation, as it allows to compute $n$ under a modulo.
 
+In group theory, $\phi(n)$ is the [order of the multiplicative group mod n](https://en.wikipedia.org/wiki/Multiplicative_group_of_integers_modulo_n) $(\mathbb Z / n\mathbb Z)^\times$, that is the group of multiplicative units (elements with inverses). The elements with multiplicative inverses are precisely those coprime to $n$.
```

Comment:
```suggestion
In group theory, $\phi(n)$ is the [order of the multiplicative group mod n](https://en.wikipedia.org/wiki/Multiplicative_group_of_integers_modulo_n) $(\mathbb Z / n\mathbb Z)^\times$, that is the number of elements with inverses. The elements with multiplicative inverses are precisely those coprime to $n$.
```

I've never heard of "multiplicative units", and $\phi(n)$ is not a group although you could form one from them.

Reply:
They can just be called units. The group I'm referring to is Z/nZ* but it is a little unclearly worded.